### DISCUSSION TITLE
Diary-中国古动物馆 | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#309] [Announcements]
# Diary-中国古动物馆 | Zi-Zi's Journey

给我约上了！

https://gz-metal-cell.github.io/2024/01/22/Diary-%E4%B8%AD%E5%9B%BD%E5%8F%A4%E5%8A%A8%E7%89%A9%E9%A6%86/

**Comment by promefire** [comment]
明年我也去 :sob: :sob: :sob:

**Reply by GZ-Metal-Cell** [reply]
确实还是不错的🤔但我觉得还是海博馆更牛逼
